web-app/src/classes/Match.ts

```diff
@@ -0,0 +1,17 @@
+import { Match as IMatch, MatchSide } from 'app/../shared/types/Tournament';
+
+export default class Match implements IMatch {
+  id?: string | undefined;
+  sideA: MatchSide;
+  sideB: MatchSide;
+  isFinished: boolean;
+  // date: Date;
+
+  constructor(data: IMatch) {
+    this.id = data.id;
+    this.sideA = data.sideA;
+    this.sideB = data.sideB;
+    this.isFinished = data.isFinished;
+    // this.date = data.date;
+  }
+}
```

Comment:
And this class is for? Can't we just make object that's type is Match (from sharedTypes?) and don't create this extra class which makes everything just less understandable?

Reply:
This is trash, useless. I forgot to delete this